drake/systems/trajectories/piecewise_polynomial_trajectory.h

```diff
@@ -13,12 +14,31 @@ namespace drake {
  */
 class PiecewisePolynomialTrajectory : public Trajectory {
  public:
+  /**
+   * Construct a PiecewisePolynomialTrajectory from a trajectory matrix.
```

Comment:
@lucy-tri -- At the very least, this needs more documentation.  There are many ways to construct a piecewise polynomial trajectory from this input data.  It looks like you've implemented a first-order hold here.  

see:
https://github.com/RobotLocomotion/drake/blob/f72bb3f465f69f25459a7a65c57b45c795b5e31d/drake/matlab/util/zoh.m
https://github.com/RobotLocomotion/drake/blob/f72bb3f465f69f25459a7a65c57b45c795b5e31d/drake/matlab/util/foh.m
https://github.com/RobotLocomotion/drake/blob/f72bb3f465f69f25459a7a65c57b45c795b5e31d/drake/matlab/util/pchipDeriv.m
and
https://www.mathworks.com/help/matlab/ref/spline.html

it would be really nice to have all of those options available here, too.